ebtables/ebtables.go

```diff
@@ -51,6 +52,61 @@ func SetArpReply(ipAddress net.IP, macAddress net.HardwareAddr, action string) e
 	return executeShellCommand(command)
 }
 
+// SetBrouteAccept sets an EB rule.
+func SetBrouteAccept(ipAddress, action string) error {
+	command := fmt.Sprintf(
+		"ebtables -t broute %s BROUTING --ip-dst %s -p IPv4 -j redirect --redirect-target ACCEPT",
+		action, ipAddress)
+
+	return executeShellCommand(command)
+}
+
+// GetEbtableRules gets EB rules for a table and chain.
+func GetEbtableRules(tableName, chainName string) ([] string, error) {
+	command := fmt.Sprintf(
+		"ebtables -t %s -L %s --Lmac2",
+		tableName, chainName)
+	out, err := platform.ExecuteCommand(command)
+	if err != nil {
+		return nil, err
+	}
+
+	// Splits lines and finds rules.
+	lines := strings.Split(out, "\n")
+	chainTitle := fmt.Sprintf("Bridge chain: %s", chainName)
+	var inChain bool
```

Comment:
leave blank.
 also move this var to top

var (
 inChain bool
rules []string
)
)